.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: New Command request
+about: Ask for a new command
+title: 'New Command'
+labels: 'enhancement'
+
+---
+
+| Questions         | Answers
+| ----------------- | -------------------------------------------------------
+| Command name?      | the command name "fop:domain:mycommand"
+| Command domain?  | the command domain ( configuration / images / cache / dev ... ) 
+| Command arguments?   | List the command arguments
+| Command options?   | List the command options 
+| Command return? | List what the command will return <br > success : _what command retrun in success_ <br /> error : _what command return in error_ <br> void if nothing
+| Command description?   | Describe what the command will do
```

Comment:
I would put that in second position, just after the name..

Reply:
The change is done :smiley: